Foundation/FileManager.swift

```diff
@@ -1385,25 +1388,27 @@ extension FileManager {
                     
                     // Skip the root.
                     _current = fts_read(stream)
-                    
                 }
 
                 _current = fts_read(stream)
                 while let current = _current {
+                    let filename = NSString(bytes: current.pointee.fts_path, length: Int(strlen(current.pointee.fts_path)), encoding: String.Encoding.utf8.rawValue)!._swiftObject
+                    let skipHidden = (_options.contains(.skipsHiddenFiles) && filename[filename._startOfLastPathComponent] == ".")
```

Comment:
Nit on the variable name: I expected it to just have the value of `_options.contains(.skipsHiddenFiles)`. I'd suggest `skipBecauseHidden` or something like that. But see suggested refactoring down below.